tests/integrationv2/configuration.py

```diff
@@ -95,3 +95,13 @@
 
 # List of ports available to tests.
 available_ports = AvailablePorts()
+
+def get_keyshare_flag(curve):
```

Comment:
This function should be a method in the S2N provider because it is only applicable to S2N.